paper-server/patches/sources/net/minecraft/world/level/storage/loot/LootTable.java.patch

```diff
@@ -20,19 +20,20 @@
                      output.accept(itemStack);
                  } else {
                      int count = itemStack.getCount();
-@@ -141,9 +_,22 @@
+@@ -141,9 +_,21 @@
      }
  
      public void fill(Container container, LootParams params, long seed) {
+-        LootContext lootContext = new LootContext.Builder(params).withOptionalRandomSeed(seed).create(this.randomSequence);
 +        // CraftBukkit start
-+        this.fillInventory(container, params, seed, false);
++        this.fillInventory(container, params, RandomSource.create(seed), false);
```

Comment:
This is not completely equivalent, a seed of 0 still produce random loots.

Reply:
Ah, thanks for pointing this out! :) I believe I should have fixed that now.

Reply:
Thanks can you rename the method to restore the overload (before it made sense in 77fd87e4d053a82b1a733a773b235c25c2a6c6a7) if you don't mind?

Reply:
Done, though I am not sure if that's a good idea since it generates more diff?

Reply:
Yeah sorry i meant on the other side fillInventory -> fill

Reply:
Oh sorry, that makes more sense, should be fixed now.

Should I mark the RandomSource field on the fill() with the plugin boolean as Nullable? Intellij shows me an warning that the field is NotNull but not sure if it matters or not since it's implementation code

Reply:
That's up to you, but then you will just move the warning to withOptionalRandomSource which is not null too

Reply:
Hm yeah right, will just keep it as is